src/components/chart/ha-chart-base.ts

```diff
@@ -471,11 +426,11 @@ export class HaChartBase extends LitElement {
       ...(this.plugins || []),
       {
         id: "resizeHook",
-        resize: (chart) => {
-          const change = chart.height - (this._chartHeight ?? 0);
-          if (!this._chartHeight || change > 12 || change < -12) {
-            // hysteresis to prevent infinite render loops
-            this._chartHeight = chart.height;
+        resize: (chart: Chart) => {
+          if (!this.height) {
+            // lock the height
+            // this removes empty space below the chart
+            this.height = chart.height;
```

Comment:
We should also add back the hysteresis then?

Reply:
Not needed anymore as we don't animate it. Only lock it once after the first render